test/test_rocrand_generate.cpp

```diff
@@ -69,18 +69,23 @@ TEST_P(rocrand_generate_tests, int_test)
 
 TEST_P(rocrand_generate_tests, char_test)
 {
+    const size_t size = 12563;
+    unsigned char * data = NULL;
```

Comment:
```suggestion
    unsigned char * data = nullptr;
```

Idem for all other uses of `NULL`.